docs/processes/release/tasks/publish/index.md

```diff
@@ -357,7 +357,9 @@ This step publishes pre-processed Unicode property data, which may be ingested b
 
 *   Using the output from the build bots:
     *   Navigate to the Azure Pipelines artifacts as described in the previous section
-    *   Download the files named *icuexportdata_uprops_full.zip* and *icuexportdata_uprops_sample.zip*
+    *   Download the files named *icuexport.zip*
```

Comment:
Nit: I think I prefer *icuexportdata.zip* because:

1. It's the same as the name of the tool so it's easier to see what it is
2. There's otherwise not much of an indication that it contains data files. An "export" could be code, docs, etc.

Reply:
^ ping